NSTTutorials/ResponseLoggerInjections/README.md

```diff
@@ -0,0 +1,45 @@
+[_Return to Tutorials Overview_](../README.md)
+
+# Response Logger Injections
+
+### Overview
+
+For use cases where the mock response files / data needs to be modified in some way before the file is generated, we can leverage response loggers.
+
+### Topics Covered
+
+1. [Modifying response data](#modifying-response-data)
+
+### References
+- [ResponseLoggerInjector](../../NST/src/main/java/com/ebay/service/logger/injection/ResponseLoggerInjector.java)
+
+## Modifying response data
+
+An example use case for when we would want to modify response data using the ResposneLoggerInjector could be some scenario in which you would want the fully validated mock response, but then wanted to modify some specific field’s **VALUE** to something else, for testing purposes.
```

Comment:
You may use an example of a URL to launch a web flow and you want to instead load the redirect URL that signals the web view to close. That may help people understand the use case.

Reply:
Done in 3d43f0926bdcdf8d1cddbe7d5466662922cf51bb